MahApps.Metro/Behaviours/BorderlessWindowBehavior.cs

```diff
@@ -133,7 +133,8 @@ private bool _ModifyStyle(Standard.WS removeStyle, Standard.WS addStyle)
             {
                 return false;
             }
-            var dwStyle = (Standard.WS)Standard.NativeMethods.GetWindowLongPtr(this.handle, Standard.GWL.STYLE).ToInt32();
+            var intPtr = Standard.NativeMethods.GetWindowLongPtr(this.handle, Standard.GWL.STYLE);
+            var dwStyle = (Standard.WS)(uint)(Environment.Is64BitProcess ? intPtr.ToInt64() : intPtr.ToInt32());
```

Comment:
You're casting the result of ToInt64() back to a uint so you'll still end up getting an OverflowException?